WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostListFragment.java

```diff
@@ -1000,16 +1000,18 @@ public void onDataLoaded(boolean isEmpty) {
                                             .commit();
                     }
 
-                    boolean isSavedPostsListShown = getPostListType() == ReaderPostListType.TAG_FOLLOWED
-                                                    && (mCurrentTag != null && mCurrentTag.isBookmarked());
-
                     // show snackbar when not in saved posts list
-                    if (isBookmarked && !isSavedPostsListShown) {
+                    if (isBookmarked && !isSavedPostListShown()) {
                         showBookmarkSnackbar();
                     }
                 }
             };
 
+    private boolean isSavedPostListShown() {
```

Comment:
I know this is not something you've created in this PR, but we might want to rename this method to `isBookmarksList` or something similar to be more consistent. Wdyt?